### DISCUSSION TITLE
Did Ya'll Just Give Up or Something?

### DISCUSSION THREAD

**Discussion by JMHBM** [LMMS/lmms#7337] [Project Management]
I don't get it. I see 247 current contributors to the LMMS project, but there hasn't been a single update in 4 years? LMMS could be so much bigger and used by a much broader and wider audience, but everyone seems to have just abandoned the project. We've all been waiting nearly a decade, maybe even longer, for VST/2/3 support, but for whatever reason, you guys blatantly refuse to add it in.

Especially now that we're in the age of generative artificial intelligence. This DAW should be exploding with new features and updates.

**Comment by Rossmaxx** [comment]
The 247 contributors is a false estimate, only around 5-15 devs are contributing currently at a regular pace. Most of those devs left the project at various times. Also, the project is not dead. We just don't want to make a release soon. We are waiting for some open PRs to get merged as of now. You don't need to wait either, grab a nightly and try it.

**Reply by Gabrielxd195** [reply]
> The 247 contributors is a false estimate, only around 5-15 devs are contributing currently at a regular pace. Most of those devs left the project at various times. Also, the project is not dead. We just don't want to make a release soon. We are waiting for some open PRs to get merged as of now. You don't need to wait either, grab a nightly and try it.

We have also had this discussion in the LMMS groups in Spanish, where many users ask the same question, but I always recommend the "Nocturnal" version of LMMS and I tell them about the great work you do with this beautiful daw. But their work will continue in the shadows as long as the "Stable" and "Beta" versions are still available in the download section, because these two versions of LMMS are obsolete. and they do not have the improvements and corrections that the night version does have. And although they are available but people leave because
"stable" or "secure", however in LMMS "stable" is obsolete.

I would like the developers to remove the "Stable" and "Beta" version from the download section and only leave the nightly version available, or at least leave it above the Stable and Beta, so as not to continue confusing users. new users.

LMMS is the only program I use where their trial versions are more stable than the one they claim to be "Stable". And for a long time it has been doing a lot of damage to their public image, because the majority of users use "Stable", even many famous content creators when they "try" LMMS they download the Stable version and are very disappointed, and do not recommend LMMS for That, ignoring the large number of improvements and the potential that the Night version does have. I do my most serious projects with "Nocturna" and when there is a failure I see how you fix it quickly, but when was the last time "Stable" and "Beta"? Greetings 

(Google Translate)

**Comment by kamkamkamkamkamkamkamkam** [comment]
> I don't get it. I see 247 current contributors to the LMMS project, but there hasn't been a single update in 4 years? LMMS could be so much bigger and used by a much broader and wider audience, but everyone seems to have just abandoned the project. We've all been waiting nearly a decade, maybe even longer, for VST/2/3 support, but for whatever reason, you guys blatantly refuse to add it in.
> 
> Especially now that we're in the age of generative artificial intelligence. This DAW should be exploding with new features and updates.

https://github.com/LMMS/lmms/commits/master
https://github.com/LMMS/lmms/pulls
did you not check here before typing all of that
all of the 4+ years of progress made on LMMS since 1.2.2 can be used in the nightly version @ https://lmms.io/download
